django/db/backends/postgresql/client.py

```diff
@@ -15,6 +15,7 @@ def runshell_db(cls, conn_params, parameters):
         host = conn_params.get('host', '')
         port = conn_params.get('port', '')
         dbname = conn_params.get('database', '')
+        options = conn_params.get('options', '')
```

Comment:
[`OPTIONS`](https://docs.djangoproject.com/en/3.1/ref/settings/#std:setting-OPTIONS) is a dictionary and should be consistent for all databases. I would add `'command_line_opts'` to `'OPTIONS'`, e.g.
```python
'OPTIONS': {
    'command_line_opts': 'default_text_search_config=simple',
}
```
This should be documented as a PostgreSQL-specific option in `ref/databases`, you will find similar sections for MySQL-specific options.

Reply:
I expected that options for DB would be applied to dbshell too, but it didn't. per your suggestion, I have to set **default_text_search_config=simple** 2 times, one for generic and one for cli. is it correct?

Reply:
True, but `OPTIONS` must be a dictionary so we could use
``` 
'OPTIONS': {
    'connection_options': 'default_text_search_config=simple',
}
```
and map it to the `'options'`  in `get_connection_params()`.

Reply:
got it, I'll make the change soon. Thanks